composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "gitonomy/gitlib",
+    "name": "iwazaru/gitlib",
```

Comment:
Sorry I didn't manage to install my fork and thought this was the problem. I won't publish it to packagist anyway, this is just a personal quick-fix for my app until a real fix is released.